### PR TITLE
Add SemaphoreCI to the continuous integration section

### PR DESCRIPTION
**Pull request by ervinb**

### CODE REVIEW

File: en/docs/_ci/semaphore.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Semaphore](https://semaphoreci.com/) has Yarn pre-installed for all
+supported Node.js versions, and no user interaction is required for the Yarn
+cache to work.
+
+To assure that your local Yarn version matches the one on Semaphore, add the
+line below to your setup commands, in Project Settings.
+
+```sh
+npm install -g yarn@<version>
```
Comment (Daniel15) [inline en/docs/_ci/semaphore.md]: Please avoid installing Yarn via npm in the documentation. There's a number of issues with installing Yarn via npm so it should be avoided.  Could you please use the installation script instead?

Comment (ervinb) [inline en/docs/_ci/semaphore.md]: Sure thing. Thanks for the speedy response!

**Comment by Daniel15** [comment]
If Semaphore CI use Debian or Ubuntu, you could try using the Debian
package and see if that works.

**Comment by ervinb** [comment]
@Daniel15 Let me know if you'd like any further changes.

Thanks!

**Comment by Daniel15** [comment]
Thanks!

**Pull request merged**